frontend/src/index.css

```
@@ -1,6 +1,74 @@
+/* Be Vietnam Pro - per plan Tuần 1 Ngày 3, chosen for full Vietnamese
+   diacritic coverage. Loaded before Tailwind so `font-sans` (tailwind.config.js)
+   resolves to it immediately, no flash of the fallback. */
+@import url('https://fonts.googleapis.com/css2?family=Be+Vietnam+Pro:wght@300;400;500;600;700;800&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Be+Vietnam+Pro:wght@300;400;500;600;700;800&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Be+Vietnam+Pro:wght@300;400;500;600;700;800&display=swap'" (import-notation)

(import-notation)